spec/lcm/integration/visualisation_object_transfer_spec.rb

```diff
@@ -114,4 +114,22 @@ def prepare_visualisation_object(rest_client, project)
     expect(prodVisualizationObject.first.meta['title']).to eq('Foo')
     expect(prodVisualizationObject.first.content['buckets'].first['items'].first['measure']['title']).to eq('Count of Account')
   end
+
+  after(:all) do
+    @project.delete
+    begin
```

Comment:
Similar code found in m_to_n_spec.rb:69, visualisation_object_transfer_spec.rb:120, visualisation_object_transfer_spec.rb:127 (mass = 93)